content/klc/S2.1.adoc

```diff
@@ -2,15 +2,15 @@
 brief = "General symbol naming guidelines"
 +++
 
-. Library name should not be duplicated in symbol name
+. Library naming should not be duplicated in footprint name
```

Comment:
This section (S2.1) is about symbols. Was it intentional to change `symbol` to `footprint`? Also, I like changing the original `Library name` to `The library name` rather than the current change.

Reply:
definitely a mistake.